tap4j/src/main/java/org/tap4j/parser/Tap13Parser.java

```diff
@@ -280,31 +274,24 @@ public void parseLine(String tapLine) {
             baseIndentation = indentation;
         }
 
-        if (indentation != state.getIndentationLevel()) { // indentation changed
+        StreamStatus prevState = null;
+        if (indentation != state.getIndentationLevel() && enableSubtests) { // indentation changed
 
             if (indentation > state.getIndentationLevel()) {
-                StreamStatus parentState = state;
+                int prevIndent = state.getIndentationLevel();
                 pushState(indentation); // make room for children
-
-                TapElement lastParentElement = parentState.getLastParsedElement();
-                if (lastParentElement instanceof TestResult) {
-                    final TestResult lastTestResult = (TestResult) lastParentElement;
-                    // whatever test set comes should be attached to parent
-                    if (lastTestResult.getSubtest() == null && this.enableSubtests) {
-                        lastTestResult.setSubtest(state.getTestSet());
-                        state.attachedToParent = true;
-                    }
-                }
+                if (indentation - prevIndent > 4)
```

Comment:
Could we use `if (indentation - prevIndent > 0)` instead? If I understand it right, we are changing the state only when we find a statement with 4 spaces more than the previous statement? Not sure if that's defined somewhere, but I suspect there might be tools/devs doing 2 spaces, or 3...